services/services.rst

```diff
@@ -70,22 +52,36 @@ Public Web Site at pfs.ipmu.jp (http)
 Public project information is at http://pfs.ipmu.jp , 
 such as list of meetings, list of publications, and instrument parameters. 
```

Comment:
meeting -> milestones or major meeting?
And instrument performance is also available in this site.

Reply:
fixed #812a527